plugins/dotnet-auth-core-identity/src/core/create-builders-services.ts

```diff
@@ -0,0 +1,34 @@
+import { CodeBlock, CsharpSupport } from "@amplication/csharp-ast";
+
+export function createBuildersServices(
+  resourceName: string,
+  builderServicesBlocks: CodeBlock[]
+): void {
+  builderServicesBlocks.push(
+    new CodeBlock({
+      code: `builder.Services.AddApiAuthentication();`,
+    })
+  );
+
+  const swaggerBuilder = builderServicesBlocks.find((builder) =>
+    builder.toString().includes("AddSwaggerGen")
+  );
+  if (!swaggerBuilder) return;
+
+  const swaggerBuilderIndex = builderServicesBlocks.indexOf(swaggerBuilder);
```

Comment:
you can use `findIndex()` directly instead of find() + indexOf() (and check if it returns -1 to check if you found it)

Reply:
@mulygottlieb what? but I need the index number to update the existing array:
<img width="763" alt="image" src="https://github.com/amplication/plugins/assets/97830649/ec4a4cc9-b094-4c2c-9524-950877bbed37">

Reply:
findIndex() gives you the index number directly instead of two operations (find + indexOf)